groovy/postInit/chemistry/inorganic_chemistry/Magnets.groovy

```diff
@@ -69,7 +69,7 @@ mods.gregtech.electrolyzer.recipeBuilder()
 
 ABS.recipeBuilder()
         .inputs(ore('dustSamarium'))
-        .inputs(ore('dustCobalt') * 5)
+        .inputs(ore('dustHighPurityCobalt') * 5)
         .fluidInputs(fluid('argon') * 300)
         .fluidOutputs(fluid('samarium_alloy') * 864)
         .blastFurnaceTemp(1800)
```

Comment:
should we make SmCo ABS recipe auto-generated by blastTemp property in SmCo material registration?

Reply:
alright, the manually-registered ABS recipe needs to be removed now that ABS recipes are auto-generated now.

Reply:
@planetme this recipe needs to be removed.